server.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const db = require('./data/db');
+
+
+
+const port = 5000;
+const server = express();
+
+
+server.use(express.json());
+        //get is requesting the information         //homies
+server.get('/', (req, res) => {
+// 1st arg: route where a resource can be interacted with
+  // 2nd arg: callback to deal with sending responses, and handling incoming data.
+    res.send('Hello from express');
+});
+//post is the method //post is to create
+
+server.get('/api/users', (req, res) => {
+        //get yhe users
+        db
+        .find()
+                .then(users => {
+                    res.json({ users });
+                })
+                .catch(error => {
+                    res.json({ error });
+                    //do something with error
+            });
+        
+        // get returns the user object with a specified id
+        server.get('/api/users/:id', (req, res) => {
+            //grab the id fromurl parameters
+            const id = req.params.id;
+
+            db
+                .findById(id)
+                .then(user => {
+                    res.json(users[0]);
```

Comment:
Why `users[0]` here? You have already identified a single `user` using `findById` -- just return that user and you should be set.